model/pytorch/dffml_model_pytorch/utils/utils.py

```diff
@@ -0,0 +1,92 @@
+import sys
+import inspect
+
+from torch import utils, nn
+from torchvision import transforms
+
+from dffml.base import BaseDataFlowFacilitatorObject
+from dffml.util.entrypoint import base_entry_point, entrypoint
+from .config import make_pytorch_config
+
+
+def create_layer(layer_dict):
+    try:
+        sequential_dict = nn.Sequential()
+        for name, layer in layer_dict.items():
+            parameters = {k: v for k, v in layer.items()}
+            layer_name = parameters.pop("name")
+
+            sequential_dict.add_module(
+                name, getattr(nn, layer_name)(**parameters)
+            )
+        return sequential_dict
+    except AttributeError:
```

Comment:
Let's use `inspect(layer, dict)` or `inspect(layer, str)` instead of `try/except`